src/include/migraphx/register_target.hpp

```diff
@@ -66,9 +65,6 @@ struct register_target_action
     }
 };
 
-template <class T>
-using auto_register_target = auto_register<register_target_action, T>;
```

Comment:
Dont remove this, We still want to provide the option to register a target with CRTP.